docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -32,6 +32,14 @@ Here are some examples of customizing the component. You can learn more about th
 ## Positioned snackbars
 
 There may be circumstances when the placement of the snackbar needs to be more flexible.
+You can control the origin of the anchor by specifying the `anchorOrigin` prop, with the following interface:
```

Comment:
I never liked the term anchorOrigin for `Snackbar`. Much prefer the 'placement' terminology from the Popper, `anchorOrigin` makes much more sense with a component like Popover where you specify an anchor.

```suggestion
You can control the position of the snackbar by specifying the `anchorOrigin` prop, with the following interface:
```